src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/collector/NodeConfigCollector.java

```diff
@@ -48,17 +50,23 @@
   private final ThreadPool_QueueCapacity threadPool_queueCapacity;
   private final Cache_Max_Size cacheMaxSize;
   private final Heap_Max heapMaxSize;
+  private final AggregateMetric fieldDataCacheSizeGroupByAggregation;
+  private final AggregateMetric shardRequestCacheSizeGroupByAggregation;
   private final int rcaPeriod;
   private int counter;
   private final HashMap<Resource, Double> configResult;
 
   public NodeConfigCollector(int rcaPeriod,
                              ThreadPool_QueueCapacity threadPool_queueCapacity,
                              Cache_Max_Size cacheMaxSize,
-                             Heap_Max heapMaxSize) {
+                             Heap_Max heapMaxSize,
+                             AggregateMetric fieldDataCacheSizeGroupByAggregation,
+                             AggregateMetric shardRequestCacheSizeGroupByAggregation) {
```

Comment:
Why did we choose to add the FieldData and Request cache usage to Node Config Cache ?
Both of these will keep changing frequently and will require frequent cache updated. Why not use the `Cache_FieldData_Size` and `Cache_Request_Size` metrics available to us ?